ash/src/extensions/ext/extended_dynamic_state.rs

```diff
@@ -0,0 +1,198 @@
+#![allow(dead_code)]
+use crate::version::{DeviceV1_0, InstanceV1_0};
+use crate::vk;
+use std::ffi::CStr;
+use std::mem;
+use std::ptr;
+
+#[derive(Clone)]
+pub struct ExtendedDynamicState {
+    handle: vk::Instance,
+    extended_dynamic_state_fn: vk::ExtExtendedDynamicStateFn,
+}
+
+impl ExtendedDynamicState {
+    pub fn new<I: InstanceV1_0, D: DeviceV1_0>(instance: &I, device: &D) -> Self {
+        let extended_dynamic_state_fn = vk::ExtExtendedDynamicStateFn::load(|name| unsafe {
+            mem::transmute(instance.get_device_proc_addr(device.handle(), name.as_ptr()))
+        });
+        Self {
+            handle: instance.handle(),
+            extended_dynamic_state_fn,
+        }
+    }
+
+    pub fn name() -> &'static CStr {
+        vk::ExtExtendedDynamicStateFn::name()
+    }
+
+    #[doc = "<https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/vkCmdSetCullModeEXT.html>"]
+    pub unsafe fn cmd_set_cull_mode(
+        &self,
+        command_buffer: vk::CommandBuffer,
+        cull_mode: vk::CullModeFlags,
+    ) {
+        self.extended_dynamic_state_fn
+            .cmd_set_cull_mode_ext(command_buffer, cull_mode);
```

Comment:
To be consistent with the functions below, drop the trailing `;` (that immediately enforces `cmd_set_cull_mode_ext` to return the same - in this case `()` - as the surrounding `fn cmd_set_cull_mode`).

Reply:
You've got an eye for these things :)

Reply:
Sometimes ^^